docs/modules/ROOT/pages/runbooks/podcapacity.adoc

```diff
@@ -0,0 +1,112 @@
+= Alert Group: syn-PodCapacity
+
+An OpenShift cluster can handle a limited number of pods per node.
+If you try to run more pods than this limit, OpenShift won't be able to schedule these pods.
+
+We provide two rules to notice if a cluster is hitting the limit of runnable pods.
+
+== Alert Rule: SYN_TooManyPods [[SYN_TooManyPods]]
+
+=== icon:glasses[] Overview
+
+This alert means that the cluster only has capacity for few additional pods.
+By default this threshold is the pod limit per node.
+
+This means if you receive this alert the cluster will likely won't be able to keep all workloads running in case of a worker node failure.
```

Comment:
```suggestion
If you receive this alert, it's likely that the cluster won't be able to keep all workloads running in case of a worker node failure.
```